Code/Tools/ProjectManager/Source/FormLineEditTagsWidget.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#if !defined(Q_MOC_RUN)
+#include <QWidget>
+#include <FormLineEditWidget.h>
+#endif
+
+class QCompleter;
+class QPushButton;
+class QLineEdit;
+class QLabel;
+class QFrame;
+class QKeyEvent;
+
+namespace AzQtComponents
+{
+    class StyledLineEdit;
+}
+
+namespace O3DE::ProjectManager
+{
+    class FormLineEditTagsWidget
+        : public FormLineEditWidget 
+    {
+     
```

Comment:
I think you need the Q_OBJECT macro on this class
> The Q_OBJECT macro must appear in the private section of a class definition that declares its own signals and slots or that uses other services provided by Qt's meta-object system.
```suggestion
     Q_OBJECT
```

Reply:
Per AMZN-alexpete's recent comments, I think the tags file portion is resolved, so I'll focus on the Q_OOBJECT macro here along with doxygen comment.